hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/util/HiveSchemaUtil.java

```diff
@@ -143,38 +145,73 @@ private static boolean isFieldExistsInSchema(Map<String, String> newTableSchema,
    * @param messageType : Parquet Schema
    * @return : Hive Table schema read from parquet file MAP[String,String]
    */
-  private static Map<String, String> convertParquetSchemaToHiveSchema(MessageType messageType, boolean supportTimestamp) throws IOException {
-    Map<String, String> schema = new LinkedHashMap<>();
+  public static Map<String, String> convertParquetSchemaToHiveSchema(MessageType messageType, boolean supportTimestamp) throws IOException {
+    return convertMapSchemaToHiveSchema(parquetSchemaToMapSchema(messageType, supportTimestamp, true));
+  }
+
+  /**
+   * Returns equivalent Hive table Field schema read from a parquet file.
+   *
+   * @param messageType : Parquet Schema
+   * @return : Hive Table schema read from parquet file List[FieldSchema] without partitionField
+   */
+  public static List<FieldSchema> convertParquetSchemaToHiveFieldSchema(MessageType messageType, HiveSyncConfig syncConfig) throws IOException {
+    return convertMapSchemaToHiveFieldSchema(parquetSchemaToMapSchema(messageType, syncConfig.supportTimestamp, false), syncConfig);
```

Comment:
@jsbali why is doFormat false here? Can we use doFormat true everywhere and simplify this code? Or does this break backward compatibiity for some sync executors? 

if we cannot make code change for backward compatibility, could you add comment why doFormat is false here?

Reply:
added it to the javadoc of the parquetSchemaToMapSchema method itself